presto-bytecode/src/main/java/com/facebook/presto/bytecode/CompilerUtils.java

```diff
@@ -48,14 +50,15 @@
     private static final boolean RUN_ASM_VERIFIER = false; // verifier doesn't work right now
     private static final AtomicReference<String> DUMP_CLASS_FILES_TO = new AtomicReference<>();
     private static final AtomicLong CLASS_ID = new AtomicLong();
+    private static final DateTimeFormatter TIMESTAMP_FORMAT = DateTimeFormat.forPattern("YYYYMMdd_HHmmss_SSS").withZoneUTC();
```

Comment:
No need for JodaTime just to format a timestamp. Use the JDK: https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html

Reply:
We only use JodaTime for execution because the API doesn’t require boxing. Otherwise the JDK is preferred.